site/client/pages/Information/Information.js

```diff
@@ -1,5 +1,11 @@
 Template.Information.rendered = function () {
   App.UI.fullScreenContainer(this.$('.full-screen-container'));
+  if (ResponsiveHelpers.isXs()) {
+    Meteor.setTimeout(function() {
+      sAlert.info('<a href="https://docs.google.com/forms/d/1qUgKrfhuAOpsmqCE-9blPOyd2m7WjeuaO7qBdDdmARM" target="_blank" class="default-color">Call for Speakers is now Open!</a>', {html: true});
```

Comment:
It isn't rotated on ios as is due to some css bug I didn't care to figure out and that didn't look very good either

Maybe if it's fixed to the top?